docs/source/algorithms/CorelliPowderCalibrationDatabase-v1.rst

```diff
@@ -0,0 +1,145 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The algorithm `CORELLIPowderCalibrationGenerate` receives (among other inputs) an `InputWorkspace` and produce and output `CalibrationTable`, 
+a `TableWorkspace` object. The contents of the calibration table patch are:
+
+
+ | ComponentName | Xposition | Yposition| Zposition| XdirectionCosine | YdirectionCosine | ZdirectionCosine | RotationAngle |
+
+
+This issue will create algorithm `CORELLIPowderCalibrationDatabase` that will receive as input `CalibrationTablePatch` and `InputWorkspace`. 
+A more detailed list of inputs:
+
+- `InputWorkspace` [Input]: `EventsWorkspace` or `MatrixWorkspace`, the run used for the calibration. The day the experiment ran will be extracted
+
+- `CalibrationTablePatch` [Input] A `TableWorkspace` object, the output of `CORELLIPowderCalibrationGenerate`
+
+- `DatabaseDirectory` [Optional Input]: absolute path to the database, with a default location to be determined by the instrument scientists.
+
+- `DayStamp` [Optional Input]: to be used in place of the `InputWorkspace`
+
+- `CalibrationTable` [Optional Output]: a `TableWorkspace`
+
+`CORELLIPowderCalibrationDatabase` creates the following outputs:  
+
+**Merged Calibration TableWorkspace**: The contents of *corelli_instrument_20202015.csv* will be loaded into a `TableWorkspace` object, the `CalibrationTable` output. It will have the same structure as `CalibrationTablePatch` but should (ideally) contain a row for every bank in the instrument
```

Comment:
change `20202015` to `YYYMMDD`, explaining that `YYYMMDD` is the day stamp of `InputWorkspace`

Reply:
I also noticed that **20202015** is not a valid time stamp.